### DISCUSSION TITLE
Laravel Voyager Analytics - “You are logged in as x@xxx.com”

### DISCUSSION THREAD

**Discussion by ghost** [thedevdojo/voyager#5347] [Q&A]
I've added my Google Analytics Client ID to my Laravel Voyager application. I created the ID in the Google developer console and gave permission to my user and verified my domain name.

Does anyone have any idea why when I return to the dashboard, I don't see any analytics; instead I see the message from the title of this post?

Laravel Version: 8.44.0

**Comment by emptynick** [comment]
Are you developing on your local machine?
If so, that's not possible iirc.
Google does not allow to get the data from "localhost" (Or any other pseudo-domain)

**Reply by ghost** [reply]
Thank for your reply,

No am not developing on a local machine, it already on a live server.

Please see image for a reference.

<img width="938" alt="Capturex" src="https://user-images.githubusercontent.com/35699210/120766573-6abd7980-c52b-11eb-87dc-e90df16a9242.PNG">
